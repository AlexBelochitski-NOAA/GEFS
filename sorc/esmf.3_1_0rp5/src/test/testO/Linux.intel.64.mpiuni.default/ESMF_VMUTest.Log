20190610 112233.241961 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112233.242215 INFO     PET0  Beginning Test, file ESMF_VMUTest.F90, line         979
20190610 112233.242232 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112233.242253 ERROR    PET0 ESMF_VM.F90 2788 ESMF_VMGet() Object being used before creation Bad Object
20190610 112233.242274 INFO     PET0  PASS  VM Get before initialization Test, ESMF_VMUTest.F90, line         987
20190610 112233.242285 ERROR    PET0 ESMF_VM.F90 5168 ESMF_VMValidate() Object being used before creation Bad Object
20190610 112233.242292 INFO     PET0  PASS  VM validate Test before valid VM, ESMF_VMUTest.F90, line         994
20190610 112233.242299 INFO     PET0  PASS  VM Get Global Test, ESMF_VMUTest.F90, line        1001
20190610 112233.242305 INFO     PET0  PASS  VM Get Test, ESMF_VMUTest.F90, line        1008
20190610 112233.245180 INFO     PET0  PASS  VM print Test, ESMF_VMUTest.F90, line        1015
20190610 112233.245263 INFO     PET0  PASS  VM validate Test, ESMF_VMUTest.F90, line        1022
20190610 112233.245280 INFO     PET0  Number of failed tests:           0
20190610 112233.245286 INFO     PET0  Ending Test, file ESMF_VMUTest.F90, line        1093
