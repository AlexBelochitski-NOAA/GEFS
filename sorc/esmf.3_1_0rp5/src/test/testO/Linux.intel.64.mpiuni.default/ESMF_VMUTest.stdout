  Beginning Test, file ESMF_VMUTest.F90, line         979
  NUMBER_OF_PROCESSORS           1
  PASS  VM Get before initialization Test, ESMF_VMUTest.F90, line         987
  PASS  VM validate Test before valid VM, ESMF_VMUTest.F90, line         994
  PASS  VM Get Global Test, ESMF_VMUTest.F90, line        1001
  PASS  VM Get Test, ESMF_VMUTest.F90, line        1008
  PASS  VM print Test, ESMF_VMUTest.F90, line        1015
  PASS  VM validate Test, ESMF_VMUTest.F90, line        1022
  Number of failed tests:           0
  Ending Test, file ESMF_VMUTest.F90, line        1093
--- ESMCI::VM::print() start ---
ESMCI::VMIdPrint:
vmID located at: 0x782760
  vmKey=0x80000000
  localID: 0
--- VMK::print() start ---
vm located at: 0xbdd060
npets = 1, mypet=0
  pth_mutex =		 0xc04860
  pth_finish_count =	 (nil)
MPI_Group_size: 1
MPI_Comm_size: 1
MPI_Comm_rank in local MPI communicator: 0
MPI_Comm_rank in MPI_COMM_WORLD: 0
MPI thread level support: 0
mpi_mutex_flag: 1
mpionly: 0
nothreadsflag: 1
  lpid[0]=0, tid[0]=0, vas[0]=0, ncpet[0]=1, cid[0][0]=0
ncores = 1
  cpuid[0]=0, ssiid[0]=0
--- VMK::print() end ---
--- ESMCI::VM::print() end ---
